output/v0.0.json

```diff
@@ -3560,6 +3575,68 @@
                     "adjustments": {
                         "type": "object",
                         "readOnly": true
+                    },
+                    "funding_drawdown_request_id": {
+                        "type": "string",
+                        "readOnly": true
+                    },
+                    "reissue_payment_method": {
+                        "$ref": "#/components/schemas/PaymentType"
+                    },
+                    "reconciliation_id": {
+                        "type": "string"
```

Comment:
Description: A string to identify the invoice in your accounting system. Max-length = 50